Resources/Prototypes/_StarLight/Entities/Objects/Specific/Mechs/equipment.yml

```
@@ -0,0 +1,24 @@
+- type: entity
+  id: MechEquipmentBorgPilot
+  parent: [ BaseMechEquipment ]
+  name: positronic interface
+  description: Allows borgs to interface with, and pilot, the mech.
```

```suggestion
  name: exosuit AI control beacon
  description: A device used to transmit exosuit data. Also allows active AI units to take control of said exosuit.
```
Tired of the same name, lets change it!

Hecc with this, allow AI Shunt too, why not!